### PR TITLE
Samuel lellis development

### PR DESCRIPTION
**Pull request by SamuelLellis**
Week 1 Snippet

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,48 +10,46 @@
             background-image: url(/images/background.jpg);
             background-size: 400% 1390%;
         }
-        bingbound{
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/index.html]: Please move these style blocks into a .css file.  And also pay attention to add an empty line between each style block

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,48 +10,46 @@
...
-        <h3> <center> Skills </center> </h3>
-        <p3> Python</br>
-        Java</br>
-        C++</br>
-        C</br>
-        </p3>
-
-
+        <h4> <center> The Road Behind </center> </h4>
+        <p4><div class = "a"> I was born and raised on Long Island. I come from a relatively </br>
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/index.html]: A new line for <div>

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,48 +10,46 @@
...
+        large family with me being the youngest of 5 siblings. Fighting with my siblings over essentials 
+        such as the last piece of chocolate cake has taught me </br> how cold and hard this world can be be.  </br>
+        I enjoy animals, anime, technology, and making a difference for the better in the lives of people around me. </br>
+         </div>
+        </p4>
 
         <h4> <center> The Road Ahead </center> </h4>
-        <p4> My decision to pursue computer science came in 12th grade after </br>
+        <p4><div class = "a"> My decision to pursue computer science came in 12th grade after </br>
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/index.html]: Try a better name for this div class to make it more meaningful :)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (SamuelLellis) [reply portfolio/src/main/webapp/index.html]: Done

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (SamuelLellis) [reply portfolio/src/main/webapp/index.html]: Done. Classes separated from text.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (SamuelLellis) [reply portfolio/src/main/webapp/index.html]: Done. Meaningful names allotted!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -19,13 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (wangmengkate) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You can remove the TODO comment here now.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -19,13 +19,24 @@
...
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Samuel!</h1>");
+    response.setContentType("text/json;");
+   // response.getWriter().println("Hello Samuel!");
```
Comment (wangmengkate) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Please delete the the commented line.  You might need to comment some code when debugging, but commented code can't be submitted or merged into master branch/production code base.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -31,15 +31,32 @@ function addRandomGreeting() {
...
-  response.then(handleResponse);
+   responsePromise.then(handleResponse);
+
+  //Using JSON
+  fetch('/data').then(response => response.json()).then((data) => {
+    
+    const indexContainer = document.getElementById('greeting-container');
+        indexContainer.innerText = '';
+
+
```
Comment (wangmengkate) [inline portfolio/src/main/webapp/script.js]: Delete this line.

Comment (SamuelLellis) [reply portfolio/src/main/webapp/script.js]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 5 above)
Comment (SamuelLellis) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done. Thanks

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (SamuelLellis) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

**Pull request merged**